jscomp/runtime/caml_float.ml

```diff
@@ -186,19 +186,14 @@ let caml_hypot_float : float ->  float -> float [@bs] = [%bs.raw {| function (x,
 |}]
 
```

Comment:
Yes, I plan on expanding the Math module, which should enable the removal of these.